community/consistency-check/src/test/java/org/neo4j/consistency/ConsistencyCheckToolTest.java

```diff
@@ -196,21 +196,30 @@ public void shouldExecuteRecoveryWhenStoreWasNonCleanlyShutdown() throws Excepti
     @Test
     public void shouldExitWhenRecoveryNeededButRecoveryFalseOptionSpecified() throws Exception
     {
+        EphemeralFileSystemAbstraction fileSystem = fs.get();
         // Given
-        createGraphDbAndKillIt();
+        File storeDir = storeDirectory.graphDbDir();
+        final GraphDatabaseService db =
+                new TestGraphDatabaseFactory().setFileSystem( fileSystem ).newImpermanentDatabase( storeDir );
+
+        try ( Transaction tx = db.beginTx() )
+        {
+            db.createNode();
+            tx.success();
+        }
+
+        EphemeralFileSystemAbstraction snapshot = fileSystem.snapshot();
+        db.shutdown();
```

Comment:
I would move piece of code in a method, it will make the test much more readable.